2016-10-24  Florian Weimer  <fweimer@redhat.com>

	PR libgcc/78064
	* unwind-c.c: Include auto-target.h.

--- libgcc/unwind-c.c	(revision 241490)
+++ libgcc/unwind-c.c	(revision 241491)
@@ -26,6 +26,7 @@ see the files COPYING3 and COPYING.RUNTI
 
 #include "tconfig.h"
 #include "tsystem.h"
+#include "auto-target.h"
 #include "unwind.h"
 #define NO_SIZE_OF_ENCODED_VALUE
 #include "unwind-pe.h"
